src/collective/taxonomy/exportimport.py

```diff
@@ -62,13 +64,14 @@ def importTaxonomy(context):
                              'default_language', 'write_permission',
                              'taxonomy_fieldset']:
                     try:
-                        result[name] = unicode(config.get('taxonomy', name), 'utf-8')
+                        result[name] = unicode(
+                            config.get('taxonomy', name), 'utf-8')
                     except ConfigParser.NoOptionError:
                         pass
 
                 for name in ['is_single_select', 'is_required']:
                     try:
-                        result[name] = config.get('taxonomy', name) == 'true' and True
+                        result[name] = config.get('taxonomy', name) == 'true' and True  # noqa: E501
```

Comment:
The `and True` here seems to be reminiscent of a time when there wasn't an explicit comparison involved – as far as I can tell it's unnecessary.

Reply:
you're right ... I'll change that